iothub/service/src/Registry/Models/AuthenticationMechanism.cs

```diff
@@ -25,7 +25,7 @@ public AuthenticationMechanism()
         }
 
         /// <summary>
-        /// Gets or sets the <see cref="SymmetricKey"/> used for Authentication
+        /// Gets or sets the <see cref="SymmetricKey"/> used for Authentication.
```

Comment:
Don't need a self link, and we can be more informal with the casing.

```suggestion
        /// Gets or sets the symmetric key used for authentication.
```